### DISCUSSION TITLE
Adjust (& sync) the playback rate/speed of the live timings window

### DISCUSSION THREAD

**Discussion by WesselKroos** [f1multiviewer/issue-tracker#581] [Feature requests]
I often watch practice sessions at 1.5x and 2x speed after they had been concluded for a few hours.
But the Live timing window always plays at the 1x speed, so I have to press the S hotkey every minute to resync it with the video streams.

It would be helpful if I can change the playback rate/speed of the Live timing window to keep it in sync.

Even more helpful would be if the S hotkey to sync the streams and Live timing does also sync the playback rate/speed.

**Comment by LapsTimeOFF** [comment]
Duplicate of #519
